rest/src/test/java/de/starwit/rest/controller/ApplicationControllerIntegrationTest.java

```diff
@@ -0,0 +1,134 @@
+package de.starwit.rest.controller;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.json.JacksonTester;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.Import;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.web.servlet.MockMvc;
+
+import de.starwit.dto.ApplicationDto;
+import de.starwit.mapper.ApplicationMapper;
+import de.starwit.mapper.EntityMapper;
+import de.starwit.mapper.FieldMapper;
+import de.starwit.persistence.entity.App;
+import de.starwit.service.impl.AppService;
+
+@WithMockUser(username = "admin", roles = { "ADMIN", "PBUSER" })
```

Comment:
Why apply both roles at the same time? A user could always have one role at a time, right?